lib/tools/adb-commands.js

```diff
@@ -958,18 +963,36 @@ methods.isDataOn = async function isDataOn () {
 };
 
 /**
- * Check the state of animation on the device under test.
+ * Check the state of animation on the device under test below:
+ *   - animator_duration_scale
+ *   - transition_animation_scale
+ *   - window_animation_scale
  *
  * @this {import('../adb.js').ADB}
  * @return {Promise<boolean>} True if at least one of animation scale settings
  *                   is not equal to '0.0'.
  */
 methods.isAnimationOn = async function isAnimationOn () {
-  let animator_duration_scale = await this.getSetting('global', 'animator_duration_scale');
-  let transition_animation_scale = await this.getSetting('global', 'transition_animation_scale');
-  let window_animation_scale = await this.getSetting('global', 'window_animation_scale');
-  return _.some([animator_duration_scale, transition_animation_scale, window_animation_scale],
-                (setting) => setting !== '0.0');
+  return (await B.all(ANIMATION_SCALE_KEYS.map(
+    async (k) => (await this.getSetting('global', k)) !== '0.0'))
+  ).includes(true);
+};
+
+/**
+ * Set animation scale with the given velue via adb shell settings command.
```

Comment:
should we also mention it's only available since Android 9?

Reply:
I haven't tried with lower ones. Let me check emulators, but they could be different from real devices. (I don't have real devices for such lower versions so cannot check real devices though)

Reply:
API level 24 and newer worked with emulators. No tests for real devices, thus potentially real devices could potentially have limitation but maybe they work.

Reply:
No error was raised, so I think addressing the version in docstring is ok as current expected usage in https://github.com/appium/appium-espresso-driver/pull/1007